extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.moat-tracking.js

```diff
@@ -1,36 +0,0 @@
-define('wikia.articleVideo.featuredVideo.moatTracking', [
```

Comment:
It's used by mobile-wiki with AE2... I know, I know... 🙈

Reply:
![screen shot 2018-11-19 at 14 19 46](https://user-images.githubusercontent.com/40768156/48709577-318bbb00-ec06-11e8-8730-5979a5e39bba.png)
We will get there 💪 🚀 !

Reply:
Not really :(
https://github.com/Wikia/mobile-wiki/blob/dev/app/modules/ads/legacyModule.js#L134